backend/danswer/dynamic_configs/store.py

```diff
@@ -64,8 +27,9 @@ def get_session(self) -> Iterator[Session]:
             session.close()
 
     def store(self, key: str, val: JSON_ro, encrypt: bool = False) -> None:
-        # The actual encryption/decryption is done in Postgres, we just need to choose
-        # which field to set
+        # Not encrypted in Redis, but encrypted in Postgres
+        self.redis_client.set(key, json.dumps(val))
```

Comment:
By default strings will be encoded as UTF-8 so this should be OK paired with the decode utf-8 on load.

Reply:
Yup ok sounds good